docs/date.md

```diff
@@ -482,22 +482,57 @@ Usage example:
 
 <!-- prettier-ignore-start -->
 ```javascript
-date = Temporal.Date.from('2019-01-31');
-other = Temporal.Date.from('2006-08-24');
-date.since(other);                           // => P4543D
-date.since(other, { largestUnit: 'years' }); // => P12Y5M7D
-other.since(date, { largestUnit: 'years' }); // => -P12Y5M7D
+earlier = Temporal.Date.from('2006-08-24');
+later = Temporal.Date.from('2019-01-31');
+earlier.until(later);                           // => P4543D
+earlier.until(later, { largestUnit: 'years' }); // => P12Y5M7D
+later.until(earlier, { largestUnit: 'years' }); // => -P12Y5M7D
 
 // If you really need to calculate the difference between two Dates in
 // hours, you can eliminate the ambiguity by explicitly choosing the
 // point in time from which you want to reckon the difference. For
 // example, using noon:
 noon = Temporal.Time.from('12:00');
-date.toDateTime(noon).since(other.toDateTime(noon), { largestUnit: 'hours' });
+earlier.toDateTime(noon).until(later.toDateTime(noon), { largestUnit: 'hours' });
   // => PT109032H
 ```
 <!-- prettier-ignore-end -->
 
+### date.**since**(_other_: Temporal.Date | object | string, _options_?: object) : Temporal.Duration
+
+**Parameters:**
+
+- `other` (`Temporal.Date` or value convertible to one): Another date since when to compute the difference.
+- `options` (optional object): An object with properties representing options for the operation.
+  The following options are recognized:
+  - `largestUnit` (optional string): The largest unit of time to allow in the resulting `Temporal.Duration` object.
+    Valid values are `'auto'`, `'years'`, `'months'`, `'weeks'`, and `'days'`.
+    The default is `'auto'`.
+  - `smallestUnit` (string): The smallest unit of time to round to in the resulting `Temporal.Duration` object.
+    Valid values are `'years'`, `'months'`, `'weeks'`, `'days'`.
+    The default is `'days'`, i.e., no rounding.
+  - `roundingIncrement` (number): The granularity to round to, of the unit given by `smallestUnit`.
+    The default is 1.
+  - `roundingMode` (string): How to handle the remainder, if rounding.
+    Valid values are `'nearest'`, `'ceil'`, `'trunc'`, and `'floor'`.
+    The default is `'nearest'`.
+
+**Returns:** a `Temporal.Duration` representing the time elapsed before `date` and since `other`.
+
+This method computes the difference between the two dates represented by `date` and `other`, optionally rounds it, and returns it as a `Temporal.Duration` object.
+If `other` is later than `date` then the resulting duration will be negative.
+
+This method does the same thing as the `Temporal.Date.prototype.until()` method, but reversed, and rounding takes place relative to `date` as an ending point instead of a starting point.
+If no rounding takes place, then the outcome of `date1.since(date2)` is the same as `date1.until(date2).negated()`.
```

Comment:
> If no rounding takes place, then the outcome of `date1.since(date2)` is the same as `date1.until(date2).negated()`.

I believe that this statement is incorrect. AFAIK, the starting point controls how month-end calculations work for variable-length months. (And, in the future, for variable-length days with ZDT.)  This means that reversing the order can yield different results.

For example, with `largestUnit: 'months'` if I start from `2019-02-28` and want to get to `2019-04-01`, then I'll add one month to get to Mar 28, and 4 days to get to April 1, yielding `P1M4D`. But if I start from April 1, then I'll subtract one month to get to March 1, and then subtract one more day to get to Feb 28, or `-P1M1D`.

Reply:
Could it be that this is only true after #993 is completed? Currently, the polyfill does the following because of `constrain`:
```
> Temporal.Date.from('2019-04-01').since('2019-02-28', {largestUnit: 'months'})
Temporal.Duration <P1M4D> {
  months: 1,
  days: 4
}
> Temporal.Date.from('2019-02-28').since('2019-04-01', {largestUnit: 'months'})
Temporal.Duration <-P1M4D> {
  months: -1,
  days: -4
}
> Temporal.Date.from('2019-02-28').add('P1M4D')
Temporal.Date <2019-04-01>
> Temporal.Date.from('2019-04-01').subtract('P1M1D')
Temporal.Date <2019-02-28>
> Temporal.Date.from('2019-04-01').subtract('P1M4D')
Temporal.Date <2019-02-28>
```
(as it stands, the starting point controls nothing except rounding, and `largestUnit` isn't rounding)

Reply:
> Could it be that this is only true after #993 is completed?

Correct.  But the docs should reflect expected behavior, right?

Reply:
I'd rather the docs change with #993, otherwise I have to write docs for every open issue in order to land a pull request for only one. But, I think it would be correct in either case to say, "With the default options, then the outcome of `date1.since(date2)` is the same as `date2.since(date1).negated()`." I'll go with that for now.

Reply:
SGTM